frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/constants.ts

```diff
@@ -2,11 +2,11 @@ export const CD_SIZE = 'size';
 export const CD_STORAGE_CLASS = 'storageClass';
 
 export const WINTOOLS_CONTAINER_NAMES = {
-  openshift: 'virtio-win-container',
-  ocp: 'virtio-win-container',
-  online: 'virtio-win-container',
-  dedicated: 'virtio-win-container',
-  azure: 'virtio-win-container',
+  openshift: 'registry.redhat.io/container-native-virtualization/virtio-win',
```

Comment:
Hmm...so this is something that ends up being injected from this console code into a VM?  It seems surprising to me this is hardcoded in the console versus being something more like a "os type" (windows|genericlinux|rhel7|rhel8|etc) flag as `virt-install` offers, and then the VM crd has a default of pulling this container image is of windows type or so.

And the lower level can be more intelligent about finding the image.

Basically I'd try to avoid having significant creation logic/defaults in the console, because we want to support "gitops" flows as well as using `oc` from the CLI, etc.

Reply:
@cgwalters I see the benefit of using the controller pattern, where the specific container image is selected at the back-end. Thanks for pointing it out.

But besides that, who can hello us understand if the code proposed here is going to work with restricted networks?